README.md

```diff
@@ -20,7 +20,7 @@ are unavailable then junctions will be used.
 ### copy(from, to, [options]) → Promise
 
 Recursively copies `from` to `to` and resolves its promise when finished. 
-If `to` already exists then the promise will be rejected with an `EEXIST`
+If `to` already exists then the promise will be rejected with an `EXIST`
```

Comment:
I think this is not a typo, see https://nodejs.org/api/errors.html#errors_common_system_errors.

Reply:
you're right